.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "python.testing.pytestArgs": [],
```

Comment:
We already have a shared launch config and it took me a bit of time to figure out how to get the unit tests to work in VSCode as a Jetbrains user. so this might be helpful for others doing development.